### PR TITLE
Jetpack sites not being disconnected

### PR DESCRIPTION
**Pull request by bsessions85**
The tests are leaving a lot of jetpacks sites and not clearing out the ones that are not reachable anymore. This runs a cleanup after the jetpack connect tests to clean up unreachable sites on the jetpackconnect user

### CODE REVIEW

File: specs-jetpack-calypso/wp-jetpack-connect-spec.js [hunk 1]
```diff
@@ -50,6 +50,11 @@ test.before( function() {
 test.describe( `Jetpack Connect: (${ screenSize })`, function() {
 	this.timeout( mochaTimeOut );
 
+	test.after( function() {
+		const jnFlow = new JetpackConnectFlow( driver, 'jetpackConnectUser' );
```
Comment (brbrr) [inline specs-jetpack-calypso/wp-jetpack-connect-spec.js]: I think we just need to use `jetpackConnectUser` in `Disconnect expired sites` block. Looks like that username parameter was lost in one of refactors.

Comment (brbrr) [reply specs-jetpack-calypso/wp-jetpack-connect-spec.js]: Yeah, I was confused by that magic too :)

Thanks for the update!

**Comment by brbrr** [comment]
:shipit:  LGTM!

**Comment by bsessions85** [comment]
@brbrr  Can you please take a look at the last commit? It was failing for mobile because the ensureSidebarMenuVisible() function, which is used quite a bit, wasn't actually doing anything. I updated to actually do a check.

**Comment by brbrr** [comment]
Looks good :)

**Pull request merged**